entrypoint.sh

```diff
@@ -2,15 +2,17 @@
 
 : ${INPUT_RETRIES:=3}
 
-INPUT_CF_API=$(echo "$1" | jq -r '.cf_api')
-INPUT_CF_USERNAME=$(echo "$1" | jq -r '.cf_username')
-INPUT_CF_PASSWORD=$(echo "$1" | jq -r '.cf_password')
-INPUT_CF_ORG=$(echo "$1" | jq -r '.cf_org')
-INPUT_CF_SPACE=$(echo "$1" | jq -r '.cf_space')
-INPUT_RETRIES=$(echo "$1" | jq -r '.retries | tonumber')
-INPUT_COMMAND=$(echo "$1" | jq -r '.command')
-
-cf8 api "$INPUT_CF_API"
+INPUT_CF_API=$(jq -r '.cf_api' <<< "$1")
+INPUT_CF_USERNAME=$(jq -r '.cf_username' <<< "$1")
+INPUT_CF_PASSWORD=$(jq -r '.cf_password' <<< "$1")
+INPUT_CF_ORG=$(jq -r '.cf_org' <<< "$1")
+INPUT_CF_SPACE=$(jq -r '.cf_space' <<< "$1")
+INPUT_RETRIES=$(jq -r '.retries | tonumber' <<< "$1")
+INPUT_COMMAND=$(jq -r '.command' <<< "$1")
+INPUT_SKIP_SSL_VALIDATION=$(jq -r '.skip_ssl_validation' <<< "$1")
```

Comment:
remove the `<<< "$1"` change here again. I had to manually revert it earlier in master back to `INPUT_CF_API=$(echo "$1" | jq -r '.cf_api')` as it is failing deployment. my bad for suggesting the change.